### PR TITLE
New `error` topic for influx sources

### PR DESCRIPTION
**Pull request by lego963**
Fixes: #108

### CODE REVIEW

File: tests/02_acceptance/21_sources_ext/test_awx_status.py [hunk 1]
```diff
@@ -62,5 +62,5 @@ async def test_trigger_empty_from_loop(awx_client, pub: Client, sub: Client):
     template_name = random_string()
     await pub.publish(awx_client._params.topic_in, f'{template_name};{1}')
     response = await sub.get(5)
-    no_template = {'source': "awx_api", "error": NO_TEMPLATE_PATTERN.format(template_name)}
+    no_template = dict(source='awx_api', error=NO_TEMPLATE_PATTERN.format(template_name))
```
[the remark this answers is not in the document]
Comment (lego963) [reply tests/02_acceptance/21_sources_ext/test_awx_status.py:65]: Pycharm suggested to do it

**Review by irina-pereiaslavskaia — APPROVED** [review]

File: delatore/sources/base.py [hunk 2]
```diff
@@ -108,3 +120,15 @@ def _same_status(new: dict, last: dict) -> bool:
...
+def _is_error_report(msg: dict) -> bool:
+    if 'status_list' in msg:
+        for status in msg['status_list']:
+            if 'error' in status:
+                return True
+    return False
+
+
+def _is_zero_status(msg: dict) -> bool:
+    return 'status_list' in msg and len(msg['status_list']) == 0
```
Comment (outcatcher) [inline delatore/sources/base.py:134]: You can replace `len(msg['status_list']) == 0` with `not msg['status_list']`

File: delatore/sources/influx.py [hunk 3]
```diff
@@ -629,22 +639,21 @@ class InfluxSourceRDSTest(InfluxSource):
 
     async def get_update(self) -> dict:
         main_metric = self._metrics[0]
-        last_value, last_time_ms, status = await self._get_status(main_metric)
+        next_last_value, last_time_ms, status = await self._get_status(main_metric)
```
Comment (outcatcher) [inline delatore/sources/influx.py:642]: M.b. rename to `previous_value`?

**Review by outcatcher — CHANGES_REQUESTED** [review]
Please change as discussed

**Pull request merged**